var/spack/repos/builtin/packages/libspatialite/package.py

```diff
@@ -14,16 +14,17 @@ class Libspatialite(AutotoolsPackage):
     homepage = "http://www.gaia-gis.it"
     url      = "http://www.gaia-gis.it/gaia-sins/libspatialite-sources/libspatialite-4.3.0a.tar.gz"
 
+    version('5.0.0', preferred=True, sha256='7b7fd70243f5a0b175696d87c46dde0ace030eacc27f39241c24bac5dfac6dac')
     # Must download manually from:
     #     https://www.gaia-gis.it/fossil/libspatialite/info/c7f67038bf06d98d
     # For instructions on the file:// below..
     #     https://github.com/spack/spack/issues/2489
     version('5.0.0.2.c7f67038bf',
-        sha256='f8100f71b769c7db066c6f938af6b00e920e4b90ac14c00a4f3ed7171565caab',
-        url="file://%s/SpatiaLite-c7f67038bf.tar.gz" % os.getcwd())
+            sha256='f8100f71b769c7db066c6f938af6b00e920e4b90ac14c00a4f3ed7171565caab',
+            url="file://%s/SpatiaLite-c7f67038bf.tar.gz" % os.getcwd())
 
     version('5.0.0-beta0', sha256='caacf5378a5cfab9b8e98bb361e2b592e714e21f5c152b795df80d0ab1da1c42')
-    version('4.3.0a', preferred=True,
+    version('4.3.0a',
             sha256='88900030a4762904a7880273f292e5e8ca6b15b7c6c3fb88ffa9e67ee8a5a499')
```

Comment:
Thoughts on removing these alpha/beta/random-commit releases now that a newer stable release is out?

Reply:
looking a git logs, proj@6: has some implicit dependency on this lib. I think these weird versions can be removed later once @citibeth  has a chance to evaluate what's going on. I suggest that we merge this PR as it is.